tests/openwisp2/sample_monitoring/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from openwisp_monitoring.monitoring.base.models import (
+    AbstractGraph,
+    AbstractMetric,
+    AbstractThreshold,
+)
+
+
+class DetailsModel(models.Model):
+    details = models.CharField(max_length=64, blank=True, null=True)
+
+
+class Metric(AbstractMetric):
+    pass
```

Comment:
see the changes in this commit and replicate them please: https://github.com/openwisp/openwisp-firmware-upgrader/pull/60/commits/604e04491ce9dc0b1108b6b39c7079f540d7ea57#diff-28e2ae300b95f10c04fcf32b14819f36R17-R46